src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/commands.py

```diff
@@ -39,6 +40,9 @@ def get_custom_sdk(custom_module, client_factory):
                          custom_command_type=get_custom_sdk('account', get_mediaservices_client))
         g.custom_command('create', 'create_mediaservice',
                          custom_command_type=get_custom_sdk('account', get_mediaservices_client))
+        g.command('check-name', 'check_name_availability')
```

Comment:
This is wrong because you have added a custom command already (see line below)

Reply:
Fixed!